.github/workflows/github.yml

```diff
@@ -236,7 +203,7 @@ jobs:
           ${{github.workspace}}/**/${{ env.VCMI_PACKAGE_FILE_NAME }}.${{ matrix.extension }}
 
     - name: Upload build
-      if: ${{ matrix.pack == 1 && (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/beta' || github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/features/')) && matrix.platform != 'msvc' }}
+      if: ${{ matrix.pack == 1 && (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/beta' || github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/features/')) && matrix.platform == 'msvc' }}
```

Comment:
FYI no artifacts are uploaded any more due to this change

Reply:
Hmm.. I get why it breaks non-Windows platforms but why msvc build fails to upload as well? And with ssh error?

Can remove this check later today

Reply:
maybe some setting in Actions configuration to not propagate the key for MSVC build? Don't have access to that to check.

Reply:
DEPLOY_RSA is repositoty secret so I think it should be available to every action?
Don't see any way to configure it anyway.